content/blog/2016/2016-11-20-what-jvm-versions-are-running-jenkins-the-return.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "What JVM versions are running Jenkins? 2016 Update!"
+tags:
+- general
+- statistics
+author: batmat
+---
+
+CAUTION: **Disclaimer:** What follows contains some opinions or statements which may not be seen as purely factual or neutral. Note that this represents by no mean the general position of the Jenkins governance board. This is solely my opinion as a contributor based on the data I gathered, and what I feel from the feedback of the community at large.
```

Comment:
Added that because I put some not-totally-neutral below and in conclusion. Not sure it's required as anyway, the author of the article is specified. Lemme know what you think about it.